transformer_engine/pytorch/softmax.py

```diff
@@ -257,6 +255,7 @@ def forward(
         self.input_in_fp16 = inp.dtype == torch.float16
         self.input_in_bf16 = inp.dtype == torch.bfloat16
         self.input_in_float16 = self.input_in_fp16 or self.input_in_bf16
+        self.attn_mask_type = attn_mask_type
```

Comment:
I suppose this is fine as a quick hack, but it would be better to pass it as an argument to `is_kernel_available`, `forward_fused_softmax`, and `forward_torch_softmax` instead of treating it as a member variable.